raiden/messages.py

```diff
@@ -488,16 +492,41 @@ class MediatedTransfer(LockedTransfer):
 
     cmdid = messages.MEDIATEDTRANSFER
 
-    def __init__(self, identifier, nonce, token, transferred_amount, recipient,
-                 locksroot, lock, target, initiator, fee=0):
+    def __init__(
+            self,
+            identifier,
+            nonce,
+            token,
+            transferred_amount,
+            recipient,
+            locksroot,
+            lock,
+            target,
+            initiator,
+            fee=0):
 
-        if nonce >= 2 ** 64:
+        if 0 > nonce >= 2 ** 64:
```

Comment:
Hmmm you appear to be correct. It's my bad for not checking this too. I thought python had some kind of magic to interpret mathematical like ranges since augusto used them a lot.

In this case @hackaugusto should we not also revisit all such range like conditions you have input. For example [these](https://github.com/raiden-network/raiden/pull/413/commits/27e0fe4e39bbcb537d6a478a4a680bd99be004a3) come to mind?